modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -89,24 +89,18 @@ See @ref videoio_overview for more information.
 */
 enum VideoCaptureAPIs {
        CAP_ANY          = 0,            //!< Auto detect == 0
-       CAP_VFW          = 200,          //!< Video For Windows (obsolete, removed)
        CAP_V4L          = 200,          //!< V4L/V4L2 capturing support
        CAP_V4L2         = CAP_V4L,      //!< Same as CAP_V4L
        CAP_FIREWIRE     = 300,          //!< IEEE 1394 drivers
        CAP_FIREWARE     = CAP_FIREWIRE, //!< Same value as CAP_FIREWIRE
        CAP_IEEE1394     = CAP_FIREWIRE, //!< Same value as CAP_FIREWIRE
        CAP_DC1394       = CAP_FIREWIRE, //!< Same value as CAP_FIREWIRE
        CAP_CMU1394      = CAP_FIREWIRE, //!< Same value as CAP_FIREWIRE
-       CAP_QT           = 500,          //!< QuickTime (obsolete, removed)
-       CAP_UNICAP       = 600,          //!< Unicap drivers (obsolete, removed)
        CAP_DSHOW        = 700,          //!< DirectShow (via videoInput)
        CAP_PVAPI        = 800,          //!< PvAPI, Prosilica GigE SDK
-       CAP_OPENNI       = 900,          //!< OpenNI (for Kinect)
-       CAP_OPENNI_ASUS  = 910,          //!< OpenNI (for Asus Xtion)
```

Comment:
> `CAP_OPENNI`

There are other leftovers:

```
$ grep -Rn CAP_OPENNI ./ | grep -v 'CAP_OPENNI_' | grep -v 'CAP_OPENNI2'
./modules/videoio/misc/gen_dict.json:3:        "CV_CAP_OPENNI",
./modules/videoio/include/opencv2/videoio/legacy/constants_c.h:40:    CV_CAP_OPENNI   =900,   // OpenNI (for Kinect)
./doc/tutorials/app/kinect_openni.markdown:61:    VideoCapture capture( CAP_OPENNI );
./doc/tutorials/app/kinect_openni.markdown:73:    VideoCapture capture(0); // or CAP_OPENNI
./doc/tutorials/app/kinect_openni.markdown:91:    VideoCapture capture( CAP_OPENNI );
```

They need to be changed / removed too.
(except may be legacy C constants)

---

**Other values should be checked too.**